### PR TITLE
updated naming convention of gateway packages

### PR DESCRIPTION
**Pull request by DiosAkhilleus**

### CODE REVIEW

File: solana/program_v2/packages/client/cli/package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@identity.com/gateway-v2-cli",
-  "version": "0.0.1",
+  "name": "@identity.com/gateway-solana-cli",
+  "version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (william-brooks) [reply solana/program_v2/packages/client/cli/package.json:3]: check this across all 3 packages

[the remark this answers is not in the document]
Comment (DiosAkhilleus) [reply solana/program_v2/packages/client/cli/package.json:3]: Will do

File: solana/program_v2/packages/client/core/src/AdminService.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ import {
   getConnectionByCluster,
 } from './lib/connection';
 import { NETWORK_SEED, GATEWAY_PROGRAM, SOLANA_MAINNET } from './lib/constants';
-import { GatewayV2 } from '@identity.com/gateway_v2-idl/src/gateway_v2';
+import { GatewayV2 } from '@identity.com/gateway-solana-idl/src/gateway_v2';
```
[the remark this answers is not in the document]
Comment (william-brooks) [reply solana/program_v2/packages/client/core/src/AdminService.ts:20]: I see this all over below, so run through it

File: solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+import { GatekeeperService } from '@identity.com/gateway-solana-client/src/GatekeeperService';
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
-import { NetworkService } from '@identity.com/gateway_v2-client/src/NetworkService';
+import { NetworkService } from '@identity.com/gateway-solana-client/src/NetworkService';
 import { Keypair, PublicKey } from '@solana/web3.js';
 import { TEST_ALT_NETWORK, TEST_NETWORK } from '../util/constants';
 import { setGatekeeperFlags } from '../util/lib';
-import { GatekeeperKeyFlags } from '@identity.com/gateway_v2-client/src/lib/constants';
+import { GatekeeperKeyFlags } from '@identity.com/gateway-solana-client/src/lib/constants';
```
Comment (william-brooks) [inline solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.ts:9]: Just to note again.. this should be exported directly, and not referenced in the folder.. dbl check if the other PR exports everything needed

File: solana/program_v2/packages/client/core/src/AdminService.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (DiosAkhilleus) [reply solana/program_v2/packages/client/core/src/AdminService.ts:20]: Hmm I thought this was the way to do it with yarn link... Happy to change it to a better way. Which PR are you referring to?

File: solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts [hunk 4]
```diff
@@ -1,9 +1,9 @@
-import { AdminService } from '@identity.com/gateway_v2-client/src/AdminService';
-import { NetworkService } from '@identity.com/gateway_v2-client/src/NetworkService';
-import { GatewayV2 } from '@identity.com/gateway_v2-idl/src/gateway_v2';
+import { AdminService } from '@identity.com/gateway-solana-client';
```
Comment (william-brooks) [inline solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts:1]: we can merge these like:
`import { AdminService, NetworkService } from '@identity.com/gateway-solana-client';`

File: solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
+import { NetworkService } from '@identity.com/gateway-solana-client';
+import { GatewayV2 } from '@identity.com/gateway-solana-idl';
 import * as anchor from '@project-serum/anchor';
 import { Keypair, LAMPORTS_PER_SOL, PublicKey } from '@solana/web3.js';
-import { airdrop } from '@identity.com/gateway_v2-client/src/lib/utils';
+import { airdrop } from '@identity.com/gateway-solana-client';
```
Comment (william-brooks) [inline solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts:6]: same as above

File: solana/program_v2/packages/tests/src/network-suite/update-network.test.ts [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
-import { AdminService } from '@identity.com/gateway_v2-client/src/AdminService';
-import { GatewayV2 } from '@identity.com/gateway_v2-idl/src/gateway_v2';
+import { AdminService } from '@identity.com/gateway-solana-client';
+import { GatewayV2 } from '@identity.com/gateway-solana-idl';
 import * as anchor from '@project-serum/anchor';
 import { expect } from 'chai';
 import * as chai from 'chai';
 import { describe } from 'mocha';
-import { NetworkKeyFlags } from '@identity.com/gateway_v2-client/src/lib/constants';
+import { NetworkKeyFlags } from '@identity.com/gateway-solana-client';
```
Comment (william-brooks) [inline solana/program_v2/packages/tests/src/network-suite/update-network.test.ts:8]: same

File: solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts (hunk 4 above)
Comment (DiosAkhilleus) [reply solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts:1]: Oh shoot missed this

File: solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts (hunk 5 above)
Comment (DiosAkhilleus) [reply solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts:6]: Will do

File: solana/program_v2/packages/tests/src/network-suite/update-network.test.ts (hunk 6 above)
Comment (DiosAkhilleus) [reply solana/program_v2/packages/tests/src/network-suite/update-network.test.ts:8]: 👍

**Review by william-brooks — APPROVED** [review]

**Pull request merged**